### DISCUSSION TITLE
Feedback for “Secure Deployment with Nginx”

### DISCUSSION THREAD

**Discussion by ival-fas** [LibreChat-AI/docs#79] [Q&A]
how to deploy without docker? 

i see the deployment nginx (option B) on this link 'https://www.librechat.ai/docs/remote/nginx', but this step need to run docker-compose.yml 

<img width="582" alt="Screenshot 2024-06-27 at 21 32 49" src="https://github.com/LibreChat-AI/librechat.ai/assets/113321872/2f52d1b1-d585-40ec-b655-28c2f1c532e3">

**Comment by fuegovic** [comment]
you can follow this for a "no docker" install guide:
https://www.librechat.ai/docs/local/npm
